src/js/footer/clientsCountryCookie/index.js

```diff
@@ -6,7 +6,7 @@ document.addEventListener("DOMContentLoaded", function () {
   const messageType = "website_status";
   async function fetchCountryAndPopulateFooter() {
     try {
-      const response = await fetch("https://www.cloudflare.com/cdn-cgi/trace");
+      const response = await fetch("https://deriv.com/cdn-cgi/trace");
```

Comment:
[Q] why don't we just fetch website status for more centralised approach
```
fetch("https://green.derivws.com/websockets/website_status?app_id=1")

```

Reply:
will look into this